dev/lint-python

```diff
@@ -17,6 +17,47 @@
 # limitations under the License.
 #
 
+VIRTUAL_ENV_DIR="build/venv"
+if [[ ! -z $VIRTUAL_ENV ]]; then
+    echo "We are in the following virtualenv: $VIRTUAL_ENV. Leaving it..."
+    # Cannot call "deactivate" from a bash script, copy/paste the code from virtualenv 'activate'
```

Comment:
I think we could maybe simplify this using the answer from http://stackoverflow.com/questions/29545624/how-to-deactivate-virtualenv-from-a-bash-script
(e.g. source or .) - what do you think?